e2e-tests/cypress/integration/appeal-submission-appeal-site-address.feature

```diff
@@ -0,0 +1,93 @@
+@wip
+Feature: Appellant provides the Appeal Site Address
+    I need to provide the appeal site address, so that all parties know which site the appeal is against.
+
+    Scenario Outline: Prospective Appellant submits a valid appeal site address
+        Given the user is prompted for the site address
+        When the user provides their appeal site address as <Address Line 1> and <Address Line 2> and <Town or City> and <County> and <Postcode>
+        Then the user is able to continue with the provided address
+        #And the user can see that their appeal has been updated with the provided address
+        Examples:
+            | Address Line 1  | Address Line 2 | Town or City | County       | Postcode  |
+            | "1 Taylor Road" | "Clifton"      | "Bristol"    | "South Glos" | "BS8 1TG" |
+            | "2 Taylor Road" | ""             | ""           | "South Glos" | "BS8 1TG" |
+            | "4 Taylor Road" | ""             | "Bristol"    | "South Glos" | "BS8 1TG" |
+            | "5 Taylor Road" | "Clifton"      | ""           | "South Glos" | "BS8 1TG" |
+
+    Scenario Outline: Prospective Appellant submits a site address without a mandatory field
+        Given the user is prompted for the site address
+        When the user provides their appeal site address as <Address Line 1> and <Address Line 2> and <Town or City> and <County> and <Postcode>
+        Then the user is informed that <reason>
+        #And the user can see that their appeal has NOT been updated with the provided address
+        Examples:
+            | Address Line 1 | Address Line 2 | Town or City | County       | Postcode  | reason                       |
+            | ""             | ""             | ""           | "South Glos" | "BS8 1TG" | "Address Line 1 is required" |
+            | "aaa"          | ""             | ""           | ""           | "BS8 1TG" | "County is required"         |
+            | "aaa"          | ""             | ""           | "South Glos" | ""        | "Postcode is required"       |
+
+    Scenario: We can handle multiple required-field failures at once
+        Given the user is prompted for the site address
+        When the user provides their appeal site address as "" and "" and "" and "" and ""
+        Then the user is informed that "Address Line 1 is required"
+        And the user is informed that "County is required"
+        And the user is informed that "Postcode is required"
+
+    Scenario: We can handle multiple required-field failures for Address Line 1 and County
```

Comment:
This could have a Scenario title like, "Prospective appellant fails to provide first address line or county"
This pattern could be used on some of the following similar scenarios.

Reply:
updated